librdmacm/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ publish_headers(rdma
   )
 publish_headers(infiniband
   ib.h
+  endian.h
   )
```

Comment:
I don't think a new public header is a very good idea.. Is it *really* necessary?

eg it seems like a really bad idea to put common names like  be16_to_cpu into a  public header.